packages/orbit-components/src/Card/CardSection/index.tsx

```diff
@@ -68,12 +69,14 @@ export default function CardSection({
       onKeyDown={onClick == null ? undefined : handleKeyDown(onClick)}
     >
       {(title != null || header != null) && expandable && (
```

Comment:
In this case, when it's `expandable` and `onClick` is provided, there are two divs with the role button, so we still have Nested interactive elements violation 😬 Can you please check?

Reply:
From my understanding, that should not happen. One can use `onClose` and `onOpen` that works for both controlled and uncontrolled version 🤷 

I will prevent the nested interactive by only assigning onClick and role button if the component is not expandable. But this feels very weird to me, to be honest.

Reply:
So you say that they should use `onClick` only when non expandable and when expandable use `onExpand` and `onClose`? Maybe we can mention that in the docs? 🤔

Reply:
Yes. I had it added to the docs. But now that I think about it further it is not incompatible 🤔 Check my approach and let me know what you think of it 🙏

Reply:
I like it, looks good now 👌